lib/hex_web/web/router.ex

```diff
@@ -157,7 +157,11 @@ defmodule HexWeb.Web.Router do
 
     Enum.map(packages, fn package ->
       version = latest_versions[package.id]
-      Map.put(package, :latest_version, version)
+      downloads = PackageDownload.package(package)
```

Comment:
This will run a query for every package on the page. It needs to be preloaded for the list of packages.